app/javascript/src/Navigation/styles/ContextSelector.scss

```diff
@@ -7,4 +7,8 @@ $lochMaraBlue: #0088ce; //pf-blue-400
 
 .PopNavigation-link > i {
   padding-right: 0.5rem;
+}
+
+.unauthorized {
+  pointer-events: none;
```

Comment:
Why not `cursor: not-allowed` ? It should be clear you can't interact
You can have `cursor: not-allowed`in the `<li>`, and `pointer-events: none` in the`<a>`.

Reply:
Good idea, not exactly the implementation I did but I added a Gif showing this alternative.